services/shhext/chat/sql_lite_persistence_test.go

```diff
@@ -98,6 +98,76 @@ func (s *SQLLitePersistenceTestSuite) TestPublicBundle() {
 	s.Equal(bundle.GetSignedPreKeys(), actualBundle.GetSignedPreKeys(), "It sets the right prekeys")
 }
 
+func (s *SQLLitePersistenceTestSuite) TestUpdatedBundle() {
+	key, err := crypto.GenerateKey()
+	s.Require().NoError(err)
+
+	actualBundle, err := s.service.GetPublicBundle(&key.PublicKey)
+	s.Require().NoError(err, "It does not return an error if the bundle is not there")
```

Comment:
Maybe message would read better as 
```suggestion
	s.Require().NoError(err, "Error was not returned even though bundle is not there")
```